app/validation/validator.py

```diff
@@ -394,3 +432,30 @@ def _parse_values(self, schema_json, parsed_key):
                 for schema_item in value:
                     if isinstance(schema_item, dict):
                         yield from self._parse_values(schema_item, parsed_key)
+
+    def _get_offset_date_value(self, answer_min_or_max):
+
+        if answer_min_or_max['value'] == 'now':
+            value = datetime.utcnow().strftime('%Y-%m-%d')
+        else:
+            value = answer_min_or_max['value']
+
```

Comment:
Just me being picking and not something you havw to do:`value = datetime.utcnow().strftime('%Y-%m-%d') if answer_min_or_max['value'] == 'now' else answer_min_or_max['value']`

Reply:
I see what your saying as it does mean a tiny bit less code, in this instance i think it's better to leave as is for readability

Reply:
In this case I think it's better as it is.

Ternary expressions in python feel clunky to me (compared to the C-style `conditional ? true_value : false_value`).

As a rule of thumb I think they're best used when they fit onto one 80 char line.

Personal preference though.